src/components/SupportRequest.js

```diff
@@ -208,6 +211,19 @@ const SupportRequest = _.flow(
             onChange: v => this.setState({ email: v })
           })
         ])]),
+        h(Fragment, [
```

Comment:
nit: This `Fragment` is unnecessary since we're not doing any computation (e.g. conditionals) on this chunk of content. We could just inline the children directly.